### DISCUSSION TITLE
You Probably Don't Need eslint-config-prettier or eslint-plugin-prettier

### DISCUSSION THREAD

**Discussion by mg-aceik** [Sitecore/jss#1762] [Ideas]
### Is your suggestion related to a problem? Please describe.

Please review and consider https://www.joshuakgoldberg.com/blog/you-probably-dont-need-eslint-config-prettier-or-eslint-plugin-prettier/ and also the latest recommendations on Eslint 7 https://typescript-eslint.io/blog/announcing-typescript-eslint-v7/

### Describe the solution you'd like

Update to the latest recommendations

**Comment by illiakovalenko** [comment]
@mg-aceik Thank you for your contribution! We will take a look at this improvement
